api/crossmodelrelations/crossmodelrelations.go

```diff
@@ -76,9 +76,9 @@ func (c *Client) RegisterRemoteRelations(relations ...params.RegisterRemoteRelat
 }
 
 // WatchRelationUnits returns a watcher that notifies of changes to the
-// units in the remote model for the relation with the given remote id.
-func (c *Client) WatchRelationUnits(remoteRelationArg params.RemoteRelationArg) (watcher.RelationUnitsWatcher, error) {
-	args := params.RemoteRelationArgs{Args: []params.RemoteRelationArg{remoteRelationArg}}
+// units in the remote model for the relation with the given remote token.
+func (c *Client) WatchRelationUnits(remoteRelationArg params.RemoteRelationDetails) (watcher.RelationUnitsWatcher, error) {
```

Comment:
Yeah fair enough, this was a separate struct and I merged them. I'll use a separate one again.